man/output_dim_for.Rd

```diff
@@ -0,0 +1,21 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/utils.R
+\name{output_dim_for}
+\alias{output_dim_for}
+\title{Network Output Dimension}
+\usage{
+output_dim_for(x, ...)
+}
+\arguments{
+\item{x}{(any)\cr
+The task.}
+
+\item{...}{(any)\cr
+Additional arguments. Not used yet.}
+}
+\description{
+Calculates the output dimension of a neural network for a given task that is expected by
+\pkg{mlr3torch}.
+For classification, this is the number of classes (unless it is a binary classification task,
+where it is 1). For regression, it is 1.
```

Comment:
for regression, this contradicts the docs above https://github.com/mlr-org/mlr3torch/pull/385/files#diff-599122382a84c6d4650caa7d1fdfad391c4825afb169b61e1a24da098c3f3e80
`\item regression, the output dimension is the same as the number of target features.`